proofs/verify.go

```diff
@@ -1,40 +0,0 @@
-package proofs
```

Comment:
I don't have a strong opinion but wondering if we actually want to delete this, might it be useful for verifying posts as part of submitPoSt messages?

cc @shannonwells

Reply:
Given the place where we're going to verify a PoSt (within the state machine), I don't see us getting positive utility out of `proofs.IsPoStValidWithVerifier`.